src/qat/purr/backends/qblox/live.py

```diff
@@ -69,7 +69,9 @@ def _execute_on_hardware(
                 for aq in aqs:
                     physical_channel = aq.physical_channel
                     dt = physical_channel.sample_time
-                    physical_channel.readout_start = aq.start * dt + aq.delay
+                    physical_channel.readout_start = aq.start * dt + (
+                        aq.delay if aq.delay else 0.0
```

Comment:
Minor but I would prefer more explicit if checks.

Reply:
like `aq.delay or 0.0` ?

Reply:
(the last comment was a joke in case that wasn't clear)

Reply:
:joy: